examples/order-manager/notification_service/notification_service.bal

```diff
@@ -21,21 +21,19 @@ import notification_service.types;
 
 configurable string LISTENING_SUBJECT = ?;
 
-listener nats:Listener subscription = new(nats:DEFAULT_URL);
-
 @nats:ServiceConfig {
     subject: LISTENING_SUBJECT
 }
-service nats:Service on subscription {
+service nats:Service on new nats:Listener(nats:DEFAULT_URL) {
```

Comment:
service "notificationService" on new nats:Listener(nats:DEFAULT_URL) 

Can't we do the above ?